suite-native/module-settings/src/tests/slice.test.ts

```diff
@@ -0,0 +1,27 @@
+import { configureStore, Store } from '@reduxjs/toolkit';
+
+import { fiatCurrencies } from '@suite-common/suite-config';
+
+import { appSettingsReducer, setColorScheme, setFiatCurrency } from '../slice';
+
+const store: Store = configureStore({
+    reducer: {
+        appSettings: appSettingsReducer,
+    },
+});
+
+describe('change color scheme', () => {
+    it('changes color scheme', () => {
+        const { payload } = store.dispatch(setColorScheme('dark'));
+        expect(payload).toEqual('dark');
+    });
+});
+
+describe('change fiat currency', () => {
+    it('changes fiat currency', () => {
+        const { payload } = store.dispatch(setFiatCurrency('czk'));
+        const fiatCurrency = payload;
+
+        expect(fiatCurrency).toEqual(fiatCurrencies[fiatCurrency].label);
```

Comment:
you are testing if payload is properly set in action creator? that seems kind of useless

Reply:
[a63b6d6](https://github.com/trezor/trezor-suite/pull/5895/commits/a63b6d68e072edad1de356f50f561c089493d1f3)